src/demo/Components/DocPage/index.tsx

```diff
@@ -73,6 +72,15 @@ const DocPageDemo = () => {
 
     updateBodyClassName(theme);
 
+    const enableDisableBookmarks = () => {
```

Comment:
Emm... What it means?
- enable/disable bookmarks
- enable bookmarks DisableControl
- <your case>